docs/api/11_invoices/update-invoice-status.mdx

```diff
@@ -0,0 +1,194 @@
+---
+sidebar_position: 2
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Update invoice status
+
+## Route
+
+```curl title="PUT"
+/api/v1/invoices/:lago_id
+```
+
+## Usage
+
+<Tabs groupId="prog-language">
+  <TabItem value="curl" label="Curl" default>
+
+  ```bash
+  LAGO_URL="https://api.getlago.com"
+  API_KEY="__YOUR_API_KEY__"
+
+  curl --location --request PUT "$LAGO_URL/api/v1/invoices/5eb02857-a71e-4ea2-bcf9-57d3a41bc6ba" \
+    --header "Authorization: Bearer $API_KEY" \
+    --header 'Content-Type: application/json' \
+    --data-raw '{
+      "invoice": {
+        "status": "succeeded"
+      }
+    }'
+  ```
+  </TabItem>
+
+  <TabItem value="python" label="Python">
+
+  ```python
+  from lago_python_client import Client
+  from lago_python_client.models import InvoiceStatusChange
+
+  client = Client(api_key='__YOUR_API_KEY__')
+
+  status_change = InvoiceStatusChange(
+    lago_id="5eb02857-a71e-4ea2-bcf9-57d8885990ba",
+    status="succeeded"
+  )
+
+  client.invoices().update(status_change)
+  ```
+  </TabItem>
+
+  <TabItem value="ruby" label="Ruby">
+
+  ```ruby
+  require 'lago-ruby-client'
+
+  client = Lago::Api::Client.new({api_key: '__YOUR_API_KEY__'})
+
+  client.invoices.update({
+      lago_id: "5eb02857-a71e-4ea2-bcf9-57d3a41bc6ba",
+      status: "code"
+  })
+  ```
+  </TabItem>
+
+  <TabItem value="javascript" label="Node.js">
+
+  ```javascript
+  import Client from 'lago-nodejs-client'
+
+  let client = new Client('__API_KEY__')
+
+  await client.updateInvoiceStatus({
+    lagoId: "5eb02857-a71e-4ea2-bcf9-57d8885990ba",
+    status: "succeeded"
+  })
+  ```
+  </TabItem>
+</Tabs>
+
+## Arguments
+
+```json
+{
+  "invoice": {
+    "status": "succeeded"
+  }
+}
+```
+
+| Attributes | Type | Description |
+|--|--|--|
+| lago_id | String &nbsp &nbsp &nbsp<Required>**Required**</Required> | Invoice unique identifier in Lago |
+| status | String &nbsp &nbsp &nbsp<Required>**Required**</Required> | Invoice status <br></br>It can be `pending`, `succeeded` or `failed` |
+
+## Responses
+
+<Tabs>
+  <TabItem value="200" label="HTTP 200" default>
+  The invoice status has been successfully updated.
+
+  Returns an [invoice](./invoice-object) object.
+  </TabItem>
+
+  <TabItem value="400" label="HTTP 400">
+
+  ```json
+  {
+    "status": 400,
+    "error": "Bad Request"
+  }
+  ```
+
+  The `invoice` json root is not present in the request body.
+  </TabItem>
+
+  <TabItem value="401" label="HTTP 401">
+
+  ```json
+  {
+    "status": 401,
+    "error": "Unauthorized"
+  }
+  ```
+
+  Access to the API endpoint is unhautorized.
```

Comment:
"Access to the API endpoint is not allowed."

Reply:
I would rather leave this one as it is since `unauthorized` is the keyword in the context of this http status code. Also, this sentence is used in all other pages so i guess it's better if we stick to the same description :) WDYT?

Reply:
Yes, makes sense. Then please just fix the typo ("unauthorized").